.github/workflows/stale.yml

```diff
@@ -0,0 +1,33 @@
+name: 'close stale issues/PRs'
+on:
+  schedule:
+    - cron: '* */6 * * *'
+  workflow_dispatch:
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@87c2b794b9b47a9bec68ae03c01aeb572ffebdb1
+        with:
+          repo-token: ${{ github.token }}
+          days-before-stale: 21
+          days-before-close: 7
+          only-labels: ""
+          operations-per-run: 100
+          remove-stale-when-updated: false
```

Comment:
Shall we set this to true?

Reply:
Good catch. I think I read this as `remove-stale-when-closed`. Confirmed that's [not the behavior](https://github.com/actions/stale/blob/87c2b794b9b47a9bec68ae03c01aeb572ffebdb1/src/IssueProcessor.ts#L235-L241).